nengo_gui/static/hotkeys.js

```diff
@@ -1,7 +1,10 @@
 Nengo.Hotkeys = function () {
     var self = this;
 
+    this.active = true;
+    
     document.addEventListener('keydown', function(ev) {
+    if (self.active) {
```

Comment:
I think I'm going to be picky here an enforce indenting.  At first glance, when looking over the code I misread things twice because of the indenting -- first I wondered how ```if (self.active)``` could ever be false when three lines up from that it was set to true, and then I stumbled over the lined-up closing braces at the end of the if block....  Indenting the contents of the AddEventListener would help.  :)